docs/source/user_guide/automl.ipynb

```diff
@@ -62,6 +62,36 @@
     "X = infer_feature_types(X, feature_types={'provider': 'categorical'})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data Checks\n",
+    "\n",
+    "Before calling `AutoMLSearch.search`, we should run some sanity checks on our data to ensure that the input data being passed will not run into some common issues before running a potentially time-consuming search. EvalML has various data checks that makes this easy. Each data check will return a collection of warnings and errors if it detects potential issues with the input data. This allows users to inspect their data to avoid confusing errors that may arise during the search process. You can learn about each of the data checks available through our [data checks guide](data_checks.ipynb) \n",
+    "\n",
+    "Here, we will run the `DefaultDataChecks` class, which contains a series of data checks that are generally useful."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml.data_checks import DefaultDataChecks\n",
+    "\n",
+    "data_checks = DefaultDataChecks(\"binary\", \"log loss binary\")\n",
+    "data_checks.validate(X, y)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since there were no warnings or errors returned, we can safely continue with the search process."
```

Comment:
This is great! Gonna be really cool when we can replace this with the data check actions :D

Reply:
Agreeeeed 🥳